python/cudf/cudf/_lib/column.pyi

```diff
@@ -24,8 +24,8 @@ class Column:
     def __init__(
         self,
         data: Optional[Buffer],
+        size,
```

Comment:
was this an intentional change(un-typing)?

Reply:
Nope - unintentional. Just fixed - thanks!